### PR TITLE
Import WSL2 information

### LINKED ISSUE #234: Mention WSL2
Perhaps it isn't mentioned because it's too obvious but it might be worth noting the possibility of using WSL2 explicitly under Up and Running with Ocaml.

### PR DESCRIPTION
**Pull request by patricoferris**
This PR:
 - Adds the best practices changes that weren't regenerated from #225 
 - Imports the changes made to include WSL2 (originally in #95, and fixes #234)

cc @tmattio -- I'm redirecting to v2 for the platform information as we haven't added anything like that yet, is that okay?

### CODE REVIEW

**Comment by tmattio** [comment]
> I'm redirecting to v2 for the platform information as we haven't added anything like that yet, is that okay?

That sounds good, let's create an issue to add this page before launch though!

File: data/tutorials/en/000_up_and_running.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.
 
 ### For Windows
 
-We use the [OCaml for Windows](https://fdopen.github.io/opam-repository-mingw/)
-installer which comes in 32bit and 64bit versions. This installer gives you
-opam and OCaml installations all in one go. It is used from within a Cygwin
-environment, but the executables produced have no dependency on Cygwin at all.
+OCaml on Windows is still a work in progress. In the meantime, if you only
+need to *run* OCaml programs on a Windows machine, then the simplest solution is to use the Windows Subsystem for Linux 2 (WSL2). WSL2 is a feature that allows Linux programs to run directly on Windows. WSL2 is substantially easier and faster to use than WSL1. Microsoft have comprehensive installation steps for [setting up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```
Comment (christinerose) [inline data/tutorials/en/000_up_and_running.md]:
```suggestion
need to *run* OCaml programs on a Windows machine, then the simplest solution is to use the Windows Subsystem for Linux 2 (WSL2). WSL2 is a feature that allows Linux programs to run directly on Windows. WSL2 is substantially easier and faster to use than WSL1. Microsoft has comprehensive installation steps for [setting up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```

File: data/tutorials/en/000_up_and_running.md [hunk 2]
```diff
@@ -66,10 +66,10 @@ sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.
...
+
+On the other hand, if you need Windows binaries then you will have to use the [OCaml for Windows](https://fdopen.github.io/opam-repository-mingw/) installer which comes in 32bit and 64bit versions. This installer gives you opam and OCaml installations all in one go. It is used from within a Cygwin environment, but the executables produced have no dependency on Cygwin at all. For a more comprehensive update on the state of OCaml on Windows, see the [OCaml on Windows](/platform/ocaml_on_windows.html) page on the old version of the site.
```
Comment (christinerose) [inline data/tutorials/en/000_up_and_running.md]:
```suggestion
On the other hand, if you need Windows binaries, you will have to use the [OCaml for Windows](https://fdopen.github.io/opam-repository-mingw/) installer which comes in 32bit and 64bit versions. This installer gives you Opam and OCaml installations all in one go. It's used from within a Cygwin environment, but the executables produced have no dependency on Cygwin at all. For a more comprehensive update on the state of OCaml on Windows, see the [OCaml on Windows](/platform/ocaml_on_windows.html) page on the old version of the site.
```

File: data/tutorials/en/000_up_and_running.md [hunk 3]
```diff
@@ -204,6 +204,8 @@ from the list. Now, help is available by hovering over symbols in your program:
 
 ![Visual Studio Code](/media/tutorials/vscode.png "")
 
+**On Windows using WSL2** you will remotely connect to your WSL2 instance from Visual Studio Code. Microsoft have a [useful blog post](https://code.visualstudio.com/blogs/2019/09/03/wsl2) covering getting WSL2 and Visual Studio Code connected.
```
Comment (christinerose) [inline data/tutorials/en/000_up_and_running.md]:
```suggestion
**On Windows using WSL2**, you will remotely connect to your WSL2 instance from Visual Studio Code. Microsoft has a [useful blog post](https://code.visualstudio.com/blogs/2019/09/03/wsl2) covering getting WSL2 and Visual Studio Code connected.
```

**Review by christinerose — CHANGES_REQUESTED** [review]

**Comment by tmattio** [comment]
Thanks @patricoferris and @christinerose 🙂

**Pull request merged**